account_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -177,12 +177,14 @@ def _log_provider_exception(
         """Both log error, and post a message on the provider record."""
         self.ensure_one()
         _logger.warning(
-            'Online Bank Statement provider "%s" failed to'
-            " obtain statement data since %s until %s"
-            % (
-                self.name,
-                statement_date_since,
-                statement_date_until,
+            _(
```

Comment:
Can't seem to find anything in the guidelines, but isn't it the best practice not to translate logging messages (as opposed to Exception messages, which are shown to the user)?

Reply:
The guidelines say messages should be translated. I like to see these messages in English too, as it helps in googling if there are any problems. But that can be done by running these procedures under a profile that has English as a language. On the other hand there maybe admins in many countries that prefer their own language or do not command the English language at all.

Reply:
OK, thanks for clarifying!

Reply:
@pedrobaeza What logging tools are not able to handle non English characters? Definitely the standard logging fully supports internationalization.

Reply:
Well, you would be surprised, specially pipelining, or remote logging. But anyway, it's not the point, logging messages always in English, please.